Makefile

```diff
@@ -8,11 +8,4 @@ build:
 test:
 	ginkgo ./pkg/...
 
```

Comment:
Please keep a target that we can use to update dependencies.

Also, please keep the vendor directory `go mod vendor`, so we make sure to keep the dependencies in tree. Just in case.

Reply:
Makes sense, I've implemented the requested changes.